components/constants/pref_names.cc

```diff
@@ -39,7 +39,6 @@ const char kReferralAttemptCount[] = "brave.referral.referral_attempt_count";
 const char kReferralHeaders[] = "brave.referral.headers";
 const char kReferralAndroidFirstRunTimestamp[] =
     "brave.referral_android_first_run_timestamp";
-const char kHTTPSEVerywhereControlType[] = "brave.https_everywhere_default";
```

Comment:
I'm confused, was this pref not actually being used to store the value of the setting? Why did it even exist?

Reply:
yep, Iooks weird, it was added in https://github.com/brave/brave-core/pull/436/files  but as I see there it was used only for settings page toggler as well, maybe @bbondy can help?

Reply:
looks like this pref was duplicating value from `brave_shields::kHTTPUpgradableResources` in content settings, when user changed the toggler state it writes to the `brave.https_everywhere_default` and also sets content setting here https://github.com/brave/brave-core/pull/436/files#diff-1e73a392540db2972505a0377b2eef71b66a860d49127a0f99bfab9df5ee84fcR189, but Shields uses only value from content setting https://github.com/brave/brave-core/blob/master/components/brave_shields/browser/brave_shields_util.cc#L612, so this pref can be removed as redundant